recipes/edffile/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "5.0.0" %}
+
+package:
+    name: edffile
+    version: {{ version }}
+
+source:
+    path: src
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - numpy  # does not compile against numpy's ABI. no 'x.x' needed
```

Comment:
I would remove it. Numpy at the build step usually means ABI dependency, but it is not uncommon in PyPI packges to import version from the module itself and hence require all the run dependencies at build time.